code/modules/power/gravitygenerator.dm

```diff
@@ -88,7 +88,7 @@
 		main_part.set_broken()
 
 /// Used to eat args
-/obj/machinery/gravity_generator/part/proc/on_update_icon(obj/machinery/gravity_generator/source, updates, updated)
+/obj/machinery/gravity_generator/part/proc/on__update_icon(obj/machinery/gravity_generator/source, updates, updated)
```

Comment:
why is there an extra `_`?

Reply:
because it broke and I didn't want to fix it